mne/viz/evoked.py

```diff
@@ -2363,7 +2363,6 @@ def click_func(
             this_evokeds = evokeds[cond]
             # skip CIs when possible; assign ci_fun first to get arg checking
             ci_fun = _get_ci_function_pce(ci, do_topo=do_topo)
-            ci_fun = ci_fun if len(this_evokeds) > 1 else None
```

Comment:
I don't know why this condition was there in the first place. I think it was just a mistake.

Reply:
turns out this *was* there for a good reason, but it just shouldn't apply if the user passes a custom CI function.